gateway-home/modules/ROOT/pages/index.adoc

```diff
@@ -63,23 +61,35 @@ MuleSoft recommends you choose Flex Gateway for high-availability and high-perfo
 | | Flex Gateway | Mule Gateway
 
 | *Summary*
-| Envoy-based API gateway to secure all APIs, Mule and non-Mule, running anywhere.
-| Java-based API gateway embedded into Mule.
+| Envoy-based API gateway to secure all APIs, Mule and non-Mule, running anywhere
+| Java-based API gateway embedded into Mule
 
 | *Tech Stack*
-| Underlying engine build upon Envoy, and FluentBit for logging.
-| Java Spring Application embedded into the Mule.
+|
+* Underlying engine built upon Envoy
+* Leverages FluentBit for logging
+| Java Spring Application embedded into Mule
 
 | *Use Case*
-| High-performant and high-availability. Secure any API anywhere.
-| Secure a Mule API "Basic Endpoint", or enable a dedicated proxy as an embedded library in a Mule. Also available as a Mule proxy application in CloudHub.
+|
+* High-performant and high-availability
```

Comment:
```suggestion
* High performant and high availability
```

Reply:
I keep coming back to that dash. If you use these as compound adjectives, then include the dash. Otherwise, no dash.

Reply:
High performance and highly available?

Sorry, I gotta let this go :>

Reply:
All good. I omitted the dashes.